src/main/java/es/tid/fiware/iot/ac/pap/PoliciesEndpoint.java

```diff
@@ -41,23 +44,33 @@
 
     private PolicyDao dao;
 
-    public PoliciesEndpoint(PolicyDao dao) {
+    private MetricRegistry metrics;
+
+    public PoliciesEndpoint(PolicyDao dao, MetricRegistry metrics) {
         this.dao = dao;
+        this.metrics = metrics;
     }
 
     @GET
     @UnitOfWork
+    @Correlator
+    @Timed
     public Response getPolicy(@Tenant String tenant,
                               @Correlator String correlator,
                               @PathParam("subject") String subject,
                               @PathParam("policyId") String policyId) {
-        
+
         LOGGER.debug("Getting policy with id [{}] for [{}] and subject [{}]", policyId, tenant, subject);
-        
+
         String id = URLEncoding.decode(policyId);
 
         Policy p = dao.loadPolicy(tenant, subject, id);
+        metrics.counter("incomingTransactionRequestSize").inc(tenant.length() +
```

Comment:
Keypass already uses a internal library for metrics, based on a jetty package. This library is good measuring time, but not total size of requests. This way, using a new coutner, we are rearusing that we could (input data, including headers), that is really near to real request size. The same for responses.